### PR TITLE
HLM-638 | Changes in verify certificate for revocation

### PR DESCRIPTION
**Pull request by ajay-beehyv**
@prasanna-egov @saiprakash-v @varadeth 

SearchCertificate doesn't need Authorisation token

### CODE REVIEW

**Comment by prasanna-egov** [comment]
Change to main branch.

**Comment by prasanna-egov** [comment]
Please resolve conflicts

**Comment by prasanna-egov** [comment]
Resolve conflicts please. This is urgent.

File: backend/vc-certification-service/src/controllers/certificate.controller.js [hunk 1]
```diff
@@ -153,10 +153,10 @@ function getRevokeBody(req) {
 
 async function verifyCertificate (req,res){
     const certificate = req.body;
-    const certificateEntityType = certificate.evidence[0].type[0];
+    const certificateEntityType = certificate.evidence.type;
```
Comment (saiprakash-v) [inline backend/vc-certification-service/src/controllers/certificate.controller.js:156]: Should support both the array data type and object data type of evidence

File: backend/vc-certification-service/src/utils/certification.utils.js [hunk 2]
```diff
@@ -1,7 +1,15 @@
 const truncateShard = (osId) => {
     return osId?.substring(2);
 }
-
+const checkIfArray = (object) => {
```
Comment (varadeth) [inline backend/vc-certification-service/src/utils/certification.utils.js:4]: name is misleading. checkIfArray should only return true or false. please refactor this name to something else

Comment (ajay-beehyv) [reply backend/vc-certification-service/src/utils/certification.utils.js]: Yeah, changed to extractFromArray

**Review by varadeth — APPROVED** [review]

**Review by prasanna-egov — APPROVED** [review]

**Pull request merged**